public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -0,0 +1,466 @@
+                                        <SurveyVersionTableReactComponent surveyVersions={surveyVersions} />
+                                     </div> :
+                                     <div className='pdf-link' style={styles.pdfLink}>
+                                        <a href={surveyPdfUrl} download>
+                                        <ClrIcon shape='file' className='is-solid' /> Download Survey as PDF
+                                        </a>
+                                     </div> }
+                </div>
+                {questions && <div className='survey-results' style={styles.surveyResults}>
+                    {questions.map((question, index) => {
+                        const key = question.conceptId + '-' + index;
+                        if (question.type === 'QUESTION') {
+                            return <div className='question-result' style={styles.questionResult} key={key}>
+                                     <div className='secondary-display' style={styles.secondaryDisplay}>
+                                        <div className='body-default'>
+                                            <SurveyQuestionReactComponent
+                                                                  searchTerm={searchWord}
+                                                                  surveyName={survey.name}
+                                                                  surveyConceptId={survey.conceptId}
+                                                                  isCopeSurvey={isCopeSurvey} question={question}
+                                                                  participantCount= {survey.participantCount}
+                                                                  versionAnalysis={surveyVersions}/>
+                                        </div>
+                                     </div>
+                            </div>;
+                        } else {
+                            return <h3 className='topic-text' key={key}>{question.questionString} style={styles.topicText}</h3>;
+                        }
+
+                    })
+                    }
+                </div>}
+            </div>
+        </section>
+        }
+        </div>
+        {!loading && searchWord && questions && questions.length === 0 ?
+        <div className='no-results' style={styles.noResults}>
+            <p className='results-heading no-results' style={styles.noResults}>
+            <span> No questions match any of the keywords: <strong style={styles.strong}>
+                    {searchWord}</strong>.
+            <span className='search-extra'> <button className='btn btn-link' onClick={() => this.handleChange('')}>Reset
+                      search</button></span>
+            </span>
+            </p>
+            <NoResultSearchComponent domainMatch={this.changeResults} searchValue={searchWord}
```

Comment:
We should call `fetchSurvey` directly here and drop the `changeResults` function.
Nit: can use an inline double-arrow function so we don't need to bind to `this`
```suggestion
            <NoResultSearchComponent domainMatch={(val) => this.fetchSurvey(val)} searchValue={searchWord}
```

Reply:
Changed this thanks.